### PR TITLE
TIDE-428 & TIDE-430

### PR DESCRIPTION
**Pull request by valendesigns**

### CODE REVIEW

File: queue/queue.go [hunk 1]
```diff
@@ -44,32 +44,16 @@ type AuditOption struct {
...
-func (mgr SqsManager) getStandardTask(endpoint, projectName, projectContent, projectSlug, sourceUrl, sourceType string, standards []string, audits *[]Audit) *Task {
-	task := &Task{
-		ResponseAPIEndpoint: endpoint,
-		Title:               projectName,
-		Content:             projectContent,
-		Slug:                projectSlug,
-		SourceURL:           sourceUrl,
-		SourceType:          sourceType,
-		RequestClient:       "wporg",
-		Visibility:          "public",
```
Comment (valendesigns) [inline queue/queue.go]: @rheinardkorf What about `RequestClient` & `Visibility`, do they have a default value that will be set somewhere else?

Comment (rheinardkorf) [reply queue/queue.go]: @valendesigns  I suppose we could make this env vars?

Comment (valendesigns) [reply queue/queue.go]: Sounds good!

Comment (rheinardkorf) [reply queue/queue.go]: this method no longer exists here... the env vars are used in main.go

**Pull request merged**